tests/test-buttercup.el

```diff
@@ -266,7 +269,7 @@
               "su1 su2"))))
 
 (describe "The `buttercup-spec-full-name' function"
-  (let (su1 su2 sp1)
+  (let (su1 su2 sp1 sp2)
```

Comment:
In my opinion, this can only work if there is 0 warnings on the file. If the file is full of warnings I should accept, I will just stop having a look at them and miss the important ones. That's why I sometimes accept a few workarounds to shut up the stupid rules of the linter. A better approach would be to disable some rules either per line or per section or per file (as with eslint for example).